script/clone_job.pl

```diff
@@ -27,6 +27,16 @@ =head1 SYNOPSIS
 
 clone_job.pl [OPTIONS] JOBID [KEY=[VALUE] ...]
 
+
+EXAMPLES:
```

Comment:
Please stick to the POD format (even if you might consider it ugly), e.g. see here
http://juerd.nl/site.plp/perlpodtut

Examples actually belong into SYNOPSIS but should be marked as code sections.